models/admin/Admin.model.js

```diff
@@ -0,0 +1,20 @@
+import mongoose, { Schema } from 'mongoose';
+import { accountStatus } from '../../constants/index.js';
+
+const ObjectId = Schema.Types.ObjectId;
+
+const adminSchema = new Schema(
+  {
+    userId: { type: ObjectId, ref: 'user', required: true },
```

Comment:
check my comment for Chef schema PR, about discriminators
https://mongoosejs.com/docs/discriminators.html

Reply:
We decided that we wouldn't use discriminators.